src/Microsoft.CodeQuality.Analyzers/UnitTests/ApiDesignGuidelines/OverrideEqualsOnOverloadingOperatorEqualsTests.Fixer.cs

```diff
@@ -4,7 +4,7 @@
 using Microsoft.CodeAnalysis;
 using Xunit;
 using VerifyCS = Test.Utilities.CSharpCodeFixVerifier<
-    Test.Utilities.EmptyDiagnosticAnalyzer, // Diagnostic is from the compiler
+    Microsoft.CodeAnalysis.Testing.EmptyDiagnosticAnalyzer, // Diagnostic is from the compiler
```

Comment:
Is this implemented in this repo now? I think I saw this in another PR this morning.

Reply:
I created it here, then moved it to roslyn-sdk. This pull request updates to the one defined in the latter, and deletes the one you reviewed this morning.